### PR TITLE
feat: support Report Issue capability for built-in module

### LINKED ISSUE #109781: Should "Report Issue" work for built-in extensions in Running Extensions view?
1. Open Running Extensions 
1. Open context menu
1. Built-in extensions have disabled menu for `Report Issue`

Screenshot (367)

### PR DESCRIPTION
**Pull request by chenjigeng**
This PR fixes #109781

### CODE REVIEW

File: src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ import { ExtensionType, IExtensionDescription } from 'vs/platform/extensions/com
 import { IOpenerService } from 'vs/platform/opener/common/opener';
 import { URI } from 'vs/base/common/uri';
 
+const builtinExtensionIssueUrl = 'https://github.com/microsoft/vscode.git';
```
Comment (isidorn) [inline src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts:17]: This should not have a `.git` suffix

File: src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts [hunk 2]
```diff
@@ -51,6 +54,9 @@ export class ReportExtensionIssueAction extends Action {
 		unresponsiveProfile?: IExtensionHostProfile
 	}): Promise<string> {
 		let baseUrl = extension.marketplaceInfo && extension.marketplaceInfo.type === ExtensionType.User && extension.description.repository ? extension.description.repository.url : undefined;
+		if (extension.description.isBuiltin) {
```
Comment (isidorn) [inline src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts:57]: Also check if baseUrl is not set. This is needed for clairty
`if (!baseUrl &&`

**Review by isidorn — CHANGES_REQUESTED** [review]

**Comment by isidorn** [comment]
Thanks for the PR. It looks good, I have tried it out.
I have left two minor comments. Once you tackle them I can merge this in.

File: src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts (hunk 2 above)
Comment (chenjigeng) [reply src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts]: done

File: src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts (hunk 1 above)
Comment (chenjigeng) [reply src/vs/workbench/contrib/extensions/electron-browser/reportExtensionIssueAction.ts]: done

**Comment by chenjigeng** [comment]
@isidorn I have dealt with these two comments, please check again

**Comment by isidorn** [comment]
This looks good. Thank you very much, merging in ☀️ 🍻

**Pull request merged**